### ISSUE TITLE
The user-defined Mitigation object sync from Workbench can't be shown correctly under Techniques

### ISSUE THREAD

**Issue by dickens88** [mitre-attack/attack-website#356]
Hi,

I defined a new Technique and related mitigration in Workbench, the configuration is like this:
![image](https://user-images.githubusercontent.com/8390209/153353283-61db2b1e-0184-4e2d-93fa-432d74a9f06c.png)

after i rebuilded website, i can find my Technique there but the mitigation is missing.
![image](https://user-images.githubusercontent.com/8390209/153353505-ef875629-8e3b-46ae-bf65-3bdf53ec847e.png)
instead, there is a sentence that i  don't know where it coming from.
```
This type of attack technique cannot be easily mitigated with preventive controls since it is based on the abuse of system features.
```
I suppose there is something wrong when website pull data from Workbench.

**Comment by IanDavila** [comment]
@dickens88 This is a bug in the website build. I've opened an issue that captures the problem (#358). Thanks for reporting this issue!

**Comment by dickens88** [comment]
@IanDavila I updated to the latest released version v3.6.0, but i found the issue is still there. the customized mitigation object still can not be shown in the right please, and the notice is all the same `This type of attack technique cannot be easily mitigated with preventive controls since it is based on the abuse of system features.` please check. thanks
